html/home.html

```diff
@@ -0,0 +1,190 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <!-- Font Awesome Icons -->
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0-beta3/css/all.min.css"
+      integrity="sha512-Fo3rlrZj/k7ujTnHg4CGR2D7kSs0v4LLanw2qksYuRlEzO+tcaEPQogQ0KaoGN26/zrn20ImR1DfuLWnOo7aBA=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
+    <!-- Google Fonts (Roboto Slab) -->
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto+Slab:wght@200;300;400;500;600;700&display=swap"
+      rel="stylesheet"
+    />
+    <!-- Shortcut Icon -->
+    <link rel="shortcut icon" href="../assets/images/logo.jpg" />
+    <!-- Main CSS File -->
+    <link rel="stylesheet" href="../css/home.css" />
+    <title>FlowX</title>
+  </head>
+  <body>
+    <section class="header">
+      <img src="" alt="" />
+      <div class="container">
+        <form action="">
+          <div class="input-parent">
+            <input type="text" placeholder="Search a Country" required />
+          </div>
+          <div class="recommends">
+            <!-- TODO: Add recommendations here HUSSAM in spans -->
+            <!-- <span>Gaza</span>
+            <span>Rafah</span>
+            <span>Hebron</span> -->
+          </div>
+        </form>
+      </div>
+      <!-- <div class="current-day">
+        <div class="left">
+          <p class="location">
+            <span class="country">Palestine</span>
+            <span class="city">Gaza</span>
+          </p>
+          <img src="http://openweathermap.org/img/wn/10d@2x.png" alt="" />
```

Comment:
Since these sections (and the other things that you already comment it) will be added in JavaScript, get rid of them completely from here

Reply:
Because the person who will create the elements from js know what elements need to be created and append them correctly to take its style.